hack/bazel-build-functests.sh

```diff
@@ -23,6 +23,13 @@ source hack/common.sh
 source hack/bootstrap.sh
 source hack/config.sh
 
+# Add the forbid-focus-container ginkgolinter flag when running in CI
+# to avoid focus containers sneaking past and being accidentally committed
+if [ -n "${CI}" ]; then
+    # jq still doesn't support in place editing of files
+    jq '.ginkgolinter.analyzer_flags |= . + {"forbid-focus-container":"true"}' nogo_config.json > nogo_config_ci.json && mv nogo_config_ci.json nogo_config.json
```

Comment:
I tried this line and it's working. I'm not sure what happen to the json colors when redirecting the jq output to a file. I would use the `-M` flag just for case. e.g.
```suggestion
    jq -M '.ginkgolinter.analyzer_flags |= . + {"forbid-focus-container":"true"}' nogo_config.json > nogo_config_ci.json && mv nogo_config_ci.json nogo_config.json
```

Reply:
After reading about it, jq default is to use color only when writing to the terminal, but not for files. So this is not needed.

Reply:
Yeah the resulting diff looked clean to me but thanks for checking.